### ISSUE TITLE
Break CIFAR-10 & CIFAR-100

### ISSUE THREAD

**Issue by accosmin** [accosmin/nano#157]
Currently the test error on CIFAR-10 is around 45% even for a large convolution network.
Need to see why is this: 
* more depth is needed?
* the stochastic optimizer does not decrease the training loss?
